.github/labeler.yml

```
@@ -86,6 +99,10 @@
   - gfx/**/*
   - sound/**/*
 
+"Scenarios":
+  - "**/scenario.*"
+  - data/json/scenarios.json
```

```suggestion
  - "**/scenario**"
```
To catch scenario_s_ and scenario_blacklis
> fd --type f | rg ".+/scenario"
./src/scenario.h
./data/mods/TEST_DATA/scenarios.json
./data/mods/Military_Professions/scenarios.json
./data/mods/MMA/scenarios.json
./src/scenario.cpp
./data/mods/Dark-Skies-Above/overrides/scenarios.json
./data/mods/Dark-Skies-Above/blacklists/scenario_blacklist.json
./data/mods/Magiclysm/scenarios.json
./data/mods/CRT_EXPANSION/scenarios/crt_scenarios.json
./data/mods/CRT_EXPANSION/scenarios/crt_classes.json
./data/mods/aftershock_exoplanet/scenarios.json
./data/mods/aftershock_exoplanet/setting_blacklists/scenario_blacklist.json
./data/mods/DinoMod/scenarios.json
./data/mods/Aftershock/scenarios.json
./data/mods/package_bionic_professions/scenarios.json
./data/mods/innawood/scenarios.json
./data/mods/deadly_bites/scenarios.json
./data/mods/innawood/scenario_whitelist.json
./data/json/scenarios.json
./lang/string_extractor/parsers/scenario.py